data/campaigns/World_Conquest/Modding.md

```diff
@@ -0,0 +1,490 @@
+# Modding World Conquest
+
+This section describes how to make eras/modifications to manipulate World Conquest elements.
+
+## How to add an era's units to World Conquest's enemy selection pool
+
+This file describes how to make your addon work well together with world conquest. In particular how to write an era that can be used together with world conquest.
```

Comment:
There's nothing wrong with this line, but combined with the next one it ends up with two sentences and one subclause that are all variants of "make your addon work with world conquest". Seems a bit repetitive.